packages/editor/src/bindings/post-meta.js

```diff
@@ -33,9 +34,12 @@ export default {
 		const updateMetaValue = ( newValue ) => {
 			setMeta( { ...meta, [ metaKey ]: newValue } );
 		};
+
 		return {
 			placeholder: metaKey,
-			useValue: [ metaValue, updateMetaValue ],
+			value: metaValue,
+			updateValue: updateMetaValue,
 		};
 	},
+	lockAttributesEditing: false,
```

Comment:
For this initial version, we need to lock the editing for the post meta source. We will probably change the default once we add the possibility to edit the value of the custom fields.